lessons/operator/05/README.md

```diff
@@ -1,261 +1,148 @@
-              "value": "us-west-1"
-            }
-          ],
-          "timestamp": 1552506033,
-          "value": 42.0
-        },
-        {
-           "name": "api_request_duration.seconds",
-           "tags": [
-             {
-               "name": "service",
-               "value": "example"
-             },
-             {
-               "name": "region",
-               "value": "us-west-1"
-             }
-           ],
-           "timestamp": 1552506033,
-           "value": 0.8273645
-        }
-      ]
-    },
-    "timestamp": 1552506033,
-    "id": "431a0085-96da-4521-863f-c38b480701e9",
-    "sequence": 1
-  }
+  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
 }
 ```
 
 </details>
 
-## Use cases
-
-Events must reference an `entity` and contain one or both of the `metrics` and service health information objects (i.e. `check` data); e.g. an event that only has `entity` and `metrics` objects (and no `check` object) is a valid Sensu event.
-
-Some common use cases for Sensu Events:
-
-- **Reporting service health information** (e.g. the result of a service check execution, including Nagios-style check scripts)
-- **Collecting metrics** using a service check or scraping a metrics API endpoint (e.g. Prometheus exporters)
-- **Endpoint discovery** and liveness monitoring (e.g. reporting the discovery of an endpoint, or additional data/metadata about a known entity)
-- **Dead mans switches** (e.g. emitting events<sup>*</sup> containing a TTL from a shell script, such as a backup script cron job)
+This generic container provides a tremendous amount of flexibility in terms of the types of observations that can be collected.
+For all the details on what kind of data events can hold, read the [Events reference documentation](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/). 
 
-_NOTE: dead man's switches are covered in more detail in [Lesson 7: Introduction to Agents & Entities](/lessons/operator/07/README.md#readme), and check TTLs are covered in more detail in [Lesson 8: Introduction to Checks](/lessons/operator/08/README.md#readme)._
+Let's try making an event manually, using some common command-line tools.
 
-> **PROTIP:** Events may reference an `entity` that does not exist in Sensu.
-> When a Sensu backend processes an event that references an `entity` that is not present in the Entities API, a "proxy" entity will be created containing the entity properties provided in the event payload.
->
-> _NOTE: entities and proxy entities are covered in more detail in [Lesson 7: introduction to Agents & Entities](/lessons/operator/07/README.md#readme) and [Lesson 12: Introduction to Proxy Entities & Proxy Checks](/lessons/operator/12/README.md#readme)._
+## EXERCISE 1: Create an Event Manually
+### Scenario
 
-## EXERCISE 1: create an event using shell commands and the Sensu Events API
+You want to explore the Events API, or want to generate an event manually using common shell tools. 
```

Comment:
One of my goals with these "Scenario" bits is that this would be real-world task-oriented snippets of how-to instruction, that could be used outside of the lesson format. So, setting "learning" as a goal doesn't seem like a realistic task that someone would be doing once they've got a Sensu environment up and running in prod, but they may want to generate an event manually from the command and learn about the events API if they were doing some adhoc thing or building a custom check/plugin/etc.